src/components/FilterableTable.react.js

```diff
@@ -0,0 +1,36 @@
+import {FilterBarActor} from '../actors/FilterBarActor';
+import {TableActor} from '../actors/TableActor';
+
+import {FilterBarStore} from '../stores/FilterBarStore';
+import {TableStore} from '../stores/TableStore';
+
+import {FilterBar} from './FilterBar/FilterBar.react';
+import {Table}  from './Table/Table.react';
+
+export class FilterableTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.id = props.filterableTableId;
+
+    this.filterBarStore = new FilterBarStore(props.filterbar);
+    this.tableStore = new TableStore(props.table);
+
+    this.tableActor = new TableActor(this.filterBarStore, this.tableStore);
+    this.filterBarActor = new FilterBarActor(this.filterBarStore, this.tableStore);
+  }
+
+  render() {
+    return (
+      <div key={this.id}>
+        <FilterBar
```

Comment:
Yeah, that's kind of the difficulty. I could probably get away with it if the pages aren't persistent and the entire table is rendered every time the filterbar changes (since it could then render the table itself).